corehq/apps/hqadmin/templates/hqadmin/hqadmin_base_report.html

```diff
@@ -5,7 +5,7 @@
 {% block title %}{% if current_page and section %}{% if current_page.title %}{{ current_page.title }} : {% endif %}{{ section.page_name }} {% else %}{{ report.title }}{% endif %}: Admin Reports{% endblock %}
 
 {% block js %}{{ block.super }}
-  {% if not requirejs_main %}
+  {% if not use_js_bundler %}
```

Comment:
How about making the template tag match this instead of being technology-specific? So, something like `js_bundle_main`?

Reply:
because the value is a boolean not the bundle string?

Reply:
I don't see how data type comes into it, maybe the question wasn't clear? I'm suggesting renaming `webpack_main`/`webpack_main_b3` to `js_bundle`/`js_bundle_b3` or `js_entry`/`js_entry_b3` so they don't reference webpack specifically, for the purpose of (maybe) doing less renaming if we switch implementations again.